### PR TITLE
Update link to Tox

### PR DESCRIPTION
**Pull request by daspecster**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ Running Tests
   *Using a Development Checkout* above).
 
 - To run the full set of gcloud-python tests on all platforms, install ``tox``
-  (http://codespeak.net/~hpk/tox/) into a system Python.  The ``tox`` console
+  (https://tox.readthedocs.org) into a system Python.  The ``tox`` console
```
Comment (tseaver) [inline CONTRIBUTING.rst]: Hmm, per https://pypi.python.org/pypi/tox, the "homepage" is https://testrun.org/tox/ (redirects to https://testrun.org/tox/latest/), which is likely a CNAME pointing to the RTD site.  I think we'd be better off echoing that, here.

Comment (daspecster) [inline CONTRIBUTING.rst]: sounds good!

**Comment by dhermes** [comment]
Thanks for finding the dead link!

How do you feel about squashing the commits into a single commit?

**Comment by daspecster** [comment]
Let me know if that's ok! @dhermes

**Comment by dhermes** [comment]
:+1:

**Pull request merged**